### ISSUE TITLE
2025-04-23, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#149]
# Today's Arxiv Papers

(1) [TTRL: Test-Time Reinforcement Learning](http://arxiv.org/abs/2504.16084v1), 2025-04-22

(2) [MR. Video: "MapReduce" is the Principle for Long Video Understanding](http://arxiv.org/abs/2504.16082v1), 2025-04-22

(3) [From Reflection to Perfection: Scaling Inference-Time Optimization for Text-to-Image Diffusion Models via Reflection Tuning](http://arxiv.org/abs/2504.16080v1), 2025-04-22

(4) [LLMs are Greedy Agents: Effects of RL Fine-tuning on Decision-Making Abilities](http://arxiv.org/abs/2504.16078v1), 2025-04-22

(5) [PHYBench: Holistic Evaluation of Physical Perception and Reasoning in Large Language Models](http://arxiv.org/abs/2504.16074v1), 2025-04-22

(6) [Automated Static Vulnerability Detection via a Holistic Neuro-symbolic Approach](http://arxiv.org/abs/2504.16057v1), 2025-04-22

(7) [Honey, I Shrunk the Language Model: Impact of Knowledge Distillation Methods on Performance and Explainability](http://arxiv.org/abs/2504.16056v1), 2025-04-22

(8) [Certified Mitigation of Worst-Case LLM Copyright Infringement](http://arxiv.org/abs/2504.16046v1), 2025-04-22

(9) [LLMs meet Federated Learning for Scalable and Secure IoT Management](http://arxiv.org/abs/2504.16032v1), 2025-04-22

(10) [LiveCC: Learning Video LLM with Streaming Speech Transcription at Scale](http://arxiv.org/abs/2504.16030v1), 2025-04-22

(11) [Benchmarking LLM for Code Smells Detection: OpenAI GPT-4.0 vs DeepSeek-V3](http://arxiv.org/abs/2504.16027v1), 2025-04-22

(12) [CAPO: Cost-Aware Prompt Optimization](http://arxiv.org/abs/2504.16005v1), 2025-04-22

(13) [Token-Aware Coding Flow: A Study with Nano Surge in Reasoning Model](http://arxiv.org/abs/2504.15989v1), 2025-04-22

(14) [From Human Memory to AI Memory: A Survey on Memory Mechanisms in the Era of LLMs](http://arxiv.org/abs/2504.15965v1), 2025-04-22

(15) [FairTranslate: An English-French Dataset for Gender Bias Evaluation in Machine Translation by Overcoming Gender Binarity](http://arxiv.org/abs/2504.15941v1), 2025-04-22

(16) [Low-Rank Adaptation of Neural Fields](http://arxiv.org/abs/2504.15933v1), 2025-04-22

(17) [Reasoning Physical Video Generation with Diffusion Timestep Tokens via Reinforcement Learning](http://arxiv.org/abs/2504.15932v1), 2025-04-22

(18) [StreamRL: Scalable, Heterogeneous, and Elastic RL for LLMs with Disaggregated Stream Generation](http://arxiv.org/abs/2504.15930v1), 2025-04-22

(19) [Language Models to Support Multi-Label Classification of Industrial Data](http://arxiv.org/abs/2504.15922v1), 2025-04-22

(20) [ViSMaP: Unsupervised Hour-long Video Summarisation by Meta-Prompting](http://arxiv.org/abs/2504.15921v1), 2025-04-22

(21) [Towards Test Generation from Task Description for Mobile Testing with Multi-modal Reasoning](http://arxiv.org/abs/2504.15917v1), 2025-04-22

(22) [Automated Bug Report Prioritization in Large Open-Source Projects](http://arxiv.org/abs/2504.15912v1), 2025-04-22

(23) [Synergizing RAG and Reasoning: A Systematic Review](http://arxiv.org/abs/2504.15909v1), 2025-04-22

(24) [Impact of Noise on LLM-Models Performance in Abstraction and Reasoning Corpus (ARC) Tasks with Model Temperature Considerations](http://arxiv.org/abs/2504.15903v1), 2025-04-22

(25) [SARI: Structured Audio Reasoning via Curriculum-Guided Reinforcement Learning](http://arxiv.org/abs/2504.15900v1), 2025-04-22

(26) [Dynamic Early Exit in Reasoning Models](http://arxiv.org/abs/2504.15895v1), 2025-04-22

(27) [Inducing Vulnerable Code Generation in LLM Coding Assistants](http://arxiv.org/abs/2504.15867v1), 2025-04-22

(28) [Exploring Cognitive and Aesthetic Causality for Multimodal Aspect-Based Sentiment Analysis](http://arxiv.org/abs/2504.15848v1), 2025-04-22

(29) [Pre-DPO: Improving Data Utilization in Direct Preference Optimization Using a Guiding Reference Model](http://arxiv.org/abs/2504.15843v1), 2025-04-22

(30) [DualOptim: Enhancing Efficacy and Stability in Machine Unlearning with Dual Optimizers](http://arxiv.org/abs/2504.15827v1), 2025-04-22

(31) [What's the Difference? Supporting Users in Identifying the Effects of Prompt and Model Changes Through Token Patterns](http://arxiv.org/abs/2504.15815v1), 2025-04-22

(32) [Insights from Verification: Training a Verilog Generation LLM with Reinforcement Learning with Testbench Feedback](http://arxiv.org/abs/2504.15804v1), 2025-04-22

(33) [A closer look at how large language models trust humans: patterns and biases](http://arxiv.org/abs/2504.15801v1), 2025-04-22

(34) [FinDER: Financial Dataset for Question Answering and Evaluating Retrieval-Augmented Generation](http://arxiv.org/abs/2504.15800v1), 2025-04-22

(35) [WALL-E 2.0: World Alignment by NeuroSymbolic Learning improves World Model-based LLM Agents](http://arxiv.org/abs/2504.15785v1), 2025-04-22

(36) [Automated Creativity Evaluation for Large Language Models: A Reference-Based Approach](http://arxiv.org/abs/2504.15784v1), 2025-04-22

(37) [TrustGeoGen: Scalable and Formal-Verified Data Engine for Trustworthy Multi-modal Geometric Problem Solving](http://arxiv.org/abs/2504.15780v1), 2025-04-22

(38) [Grounded in Context: Retrieval-Based Method for Hallucination Detection](http://arxiv.org/abs/2504.15771v1), 2025-04-22

(39) [BBAL: A Bidirectional Block Floating Point-Based Quantisation Accelerator for Large Language Models](http://arxiv.org/abs/2504.15721v1), 2025-04-22

(40) [SeaLLM: Service-Aware and Latency-Optimized Resource Sharing for Large Language Model Inference](http://arxiv.org/abs/2504.15720v1), 2025-04-22

(41) [Implementing Rational Choice Functions with LLMs and Measuring their Alignment with User Preferences](http://arxiv.org/abs/2504.15719v1), 2025-04-22

(42) [DianJin-R1: Evaluating and Enhancing Financial Reasoning in Large Language Models](http://arxiv.org/abs/2504.15716v1), 2025-04-22

(43) [Advancing Embodied Agent Security: From Safety Benchmarks to Input Moderation](http://arxiv.org/abs/2504.15699v1), 2025-04-22

(44) [The Viability of Crowdsourcing for RAG Evaluation](http://dx.doi.org/10.1145/3726302.3730093), 2025-04-22

(45) [VeriCoder: Enhancing LLM-Based RTL Code Generation through Functional Correctness Validation](http://arxiv.org/abs/2504.15659v1), 2025-04-22

(46) [Cost-Effective Text Clustering with Large Language Models](http://arxiv.org/abs/2504.15640v1), 2025-04-22

(47) [DR.FIX: Automatically Fixing Data Races at Industry Scale](http://arxiv.org/abs/2504.15637v1), 2025-04-22

(48) [Exploiting Contextual Knowledge in LLMs through V-usable Information based Layer Enhancement](http://arxiv.org/abs/2504.15630v1), 2025-04-22

(49) [CiteFix: Enhancing RAG Accuracy Through Post-Processing Citation Correction](http://arxiv.org/abs/2504.15629v1), 2025-04-22

(50) [FaceInsight: A Multimodal Large Language Model for Face Perception](http://arxiv.org/abs/2504.15624v1), 2025-04-22

(51) [Exploring the Role of Large Language Models in Cybersecurity: A Systematic Survey](http://arxiv.org/abs/2504.15622v1), 2025-04-22

(52) [AdaViP: Aligning Multi-modal LLMs via Adaptive Vision-enhanced Preference Optimization](http://arxiv.org/abs/2504.15619v1), 2025-04-22

(53) [A LoRA-Based Approach to Fine-Tuning LLMs for Educational Guidance in Resource-Constrained Settings](http://arxiv.org/abs/2504.15610v1), 2025-04-22

(54) [Research on Navigation Methods Based on LLMs](http://arxiv.org/abs/2504.15600v1), 2025-04-22

(55) [A Comprehensive Survey in LLM(-Agent) Full Stack Safety: Data, Training and Deployment](http://arxiv.org/abs/2504.15585v1), 2025-04-22

(56) [Instruction-Tuning Data Synthesis from Scratch via Web Reconstruction](http://arxiv.org/abs/2504.15573v1), 2025-04-22

(57) [A Large-scale Class-level Benchmark Dataset for Code Generation with LLMs](http://arxiv.org/abs/2504.15564v1), 2025-04-22

(58) [A Multi-Agent Framework for Automated Qinqiang Opera Script Generation Using Large Language Models](http://arxiv.org/abs/2504.15552v1), 2025-04-22

(59) [Do It For Me vs. Do It With Me: Investigating User Perceptions of Different Paradigms of Automation in Copilots for Feature-Rich Software](http://dx.doi.org/10.1145/3706598.3713431), 2025-04-22

(60) [LLM-based Semantic Augmentation for Harmful Content Detection](http://arxiv.org/abs/2504.15548v1), 2025-04-22

(61) [A Framework for Testing and Adapting REST APIs as LLM Tools](http://arxiv.org/abs/2504.15546v1), 2025-04-22

(62) [llm-jp-modernbert: A ModernBERT Model Trained on a Large-Scale Japanese Corpus with Long Context Length](http://arxiv.org/abs/2504.15544v1), 2025-04-22

(63) [Compass-V2 Technical Report](http://arxiv.org/abs/2504.15527v1), 2025-04-22

(64) [IPBench: Benchmarking the Knowledge of Large Language Models in Intellectual Property](http://arxiv.org/abs/2504.15524v1), 2025-04-22

(65) [The Bitter Lesson Learned from 2,000+ Multilingual Benchmarks](http://arxiv.org/abs/2504.15521v1), 2025-04-22

(66) [SimulS2S-LLM: Unlocking Simultaneous Inference of Speech LLMs for Speech-to-Speech Translation](http://arxiv.org/abs/2504.15509v1), 2025-04-22

(67) [In-context Ranking Preference Optimization](http://arxiv.org/abs/2504.15477v1), 2025-04-21

(68) [From Reviews to Dialogues: Active Synthesis for Zero-Shot LLM-based Conversational Recommender System](http://arxiv.org/abs/2504.15476v1), 2025-04-21

(69) [Speculative Sampling via Exponential Races](http://arxiv.org/abs/2504.15475v1), 2025-04-21

(70) [Agent for User: Testing Multi-User Interactive Features in TikTok](http://arxiv.org/abs/2504.15474v1), 2025-04-21

(71) [Emergence and Evolution of Interpretable Concepts in Diffusion Models](http://arxiv.org/abs/2504.15473v1), 2025-04-21

(72) [LAPP: Large Language Model Feedback for Preference-Driven Reinforcement Learning](http://arxiv.org/abs/2504.15472v1), 2025-04-21

(73) [Demand for LLMs: Descriptive Evidence on Substitution, Market Expansion, and Multihoming](http://arxiv.org/abs/2504.15440v1), 2025-04-21

(74) [Combating Toxic Language: A Review of LLM-Based Strategies for Software Engineering](http://arxiv.org/abs/2504.15439v1), 2025-04-21

(75) [Feeding LLM Annotations to BERT Classifiers at Your Own Risk](http://arxiv.org/abs/2504.15432v1), 2025-04-21

(76) [Trillion 7B Technical Report](http://arxiv.org/abs/2504.15431v1), 2025-04-21

(77) [TVR: Automotive System Requirement Traceability Validation and Recovery Through Retrieval-Augmented Generation](http://arxiv.org/abs/2504.15427v1), 2025-04-21

(78) [LLM-Assisted Translation of Legacy FORTRAN Codes to C++: A Cross-Platform Study](http://arxiv.org/abs/2504.15424v1), 2025-04-21

(79) [IV-Bench: A Benchmark for Image-Grounded Video Perception and Reasoning in Multimodal LLMs](http://arxiv.org/abs/2504.15415v1), 2025-04-21

(80) [Tell Me What You Know About Sexism: Expert-LLM Interaction Strategies and Co-Created Definitions for Zero-Shot Sexism Detection](http://arxiv.org/abs/2504.15392v1), 2025-04-21

(81) [KeDiff: Key Similarity-Based KV Cache Eviction for Long-Context LLM Inference in Resource-Constrained Environments](http://arxiv.org/abs/2504.15364v1), 2025-04-21

(82) [VisuLogic: A Benchmark for Evaluating Visual Reasoning in Multi-modal Large Language Models](http://arxiv.org/abs/2504.15279v1), 2025-04-21

(83) [Seeing from Another Perspective: Evaluating Multi-View Understanding in MLLMs](http://arxiv.org/abs/2504.15280v1), 2025-04-21

(84) [Stop Summation: Min-Form Credit Assignment Is All Process Reward Model Needs for Reasoning](http://arxiv.org/abs/2504.15275v1), 2025-04-21

(85) [Measuring Interest Group Positions on Legislation: An AI-Driven Analysis of Lobbying Reports](http://arxiv.org/abs/2504.15333v1), 2025-04-21

(86) [Interpretable Locomotion Prediction in Construction Using a Memory-Driven LLM Agent With Chain-of-Thought Reasoning](http://arxiv.org/abs/2504.15263v1), 2025-04-21

(87) [CRUST-Bench: A Comprehensive Benchmark for C-to-safe-Rust Transpilation](http://arxiv.org/abs/2504.15254v1), 2025-04-21

(88) [Evaluating Judges as Evaluators: The JETTS Benchmark of LLM-as-Judges as Test-Time Scaling Evaluators](http://arxiv.org/abs/2504.15253v1), 2025-04-21

(89) [MR. Guard: Multilingual Reasoning Guardrail using Curriculum Learning](http://arxiv.org/abs/2504.15241v1), 2025-04-21

(90) [A Self-Improving Coding Agent](http://arxiv.org/abs/2504.15228v1), 2025-04-21

(91) [Med-CoDE: Medical Critique based Disagreement Evaluation Framework](http://arxiv.org/abs/2504.15330v1), 2025-04-21

(92) [EvalAgent: Discovering Implicit Evaluation Criteria from the Web](http://arxiv.org/abs/2504.15219v1), 2025-04-21

(93) [Integrating Symbolic Execution into the Fine-Tuning of Code-Generating LLMs](http://arxiv.org/abs/2504.15210v1), 2025-04-21

(94) [Compute-Optimal LLMs Provably Generalize Better With Scale](http://arxiv.org/abs/2504.15208v1), 2025-04-21

(95) [Support Evaluation for the TREC 2024 RAG Track: Comparing Human versus LLM Judges](http://arxiv.org/abs/2504.15205v1), 2025-04-21

(96) [Zero-Shot, But at What Cost? Unveiling the Hidden Overhead of MILS's LLM-CLIP Framework for Image Captioning](http://arxiv.org/abs/2504.15199v1), 2025-04-21

(97) [Synergistic Weak-Strong Collaboration by Aligning Preferences](http://arxiv.org/abs/2504.15188v2), 2025-04-21

(98) [DSPO: Direct Semantic Preference Optimization for Real-World Image Super-Resolution](http://arxiv.org/abs/2504.15176v1), 2025-04-21

(99) [The Synthetic Imputation Approach: Generating Optimal Synthetic Texts For Underrepresented Categories In Supervised Classification Tasks](http://arxiv.org/abs/2504.15160v1), 2025-04-21

(100) [KGMEL: Knowledge Graph-Enhanced Multimodal Entity Linking](http://dx.doi.org/10.1145/3726302.3730217), 2025-04-21
